src/screens/wallet/wallet.tsx

```diff
@@ -3,20 +3,29 @@ import { Text, TouchableOpacity, View } from 'react-native';
 import { useDispatch } from 'react-redux';
 
 import { useBottomSheet } from '../../components/bottom-sheet/use-bottom-sheet.hook';
-import { loadAssetsActions } from '../../store/assets/assets-actions';
-import { useAssetsSelector } from '../../store/assets/assets-selectors';
+import { loadTezosAssetsActions, loadTokenAssetsActions } from '../../store/assets/assets-actions';
+import { useAssetsSelector, useTezosSelector } from '../../store/assets/assets-selectors';
 import { useFirstAccountSelector } from '../../store/wallet/wallet-selectors';
 import { ReceiveBottomSheet } from './receive-bottom-sheet/receive-bottom-sheet';
+import { SendBottomSheet } from './send-bottom-sheet/send-bottom-sheet';
 import { WalletStyles } from './wallet.styles';
 
 export const Wallet = () => {
   const firstAccount = useFirstAccountSelector();
   const assets = useAssetsSelector();
+  const tezos = useTezosSelector();
 
   const dispatch = useDispatch();
-  const { isOpen, onOpen, onClose, onDismiss } = useBottomSheet();
+  const {
+    isOpen: isOpenReceive,
+    onOpen: onOpenReceive,
+    onClose: onCloseReceive,
+    onDismiss: onDismissReceive
+  } = useBottomSheet();
+  const { isOpen: isOpenSend, onOpen: onOpenSend, onClose: onCloseSend, onDismiss: onDismissSend } = useBottomSheet();
```

Comment:
Maybe let's omit object destructuring in this case?  
 ```
 const receiveBottomSheet = useBottomSheet();
 const sendBottomSheet = useBottomSheet();
 ```
 
 what do you think?